test/e2e/serial/tests/workload_placement_tmpol.go

```diff
@@ -446,9 +446,9 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 			Expect(err).ToNot(HaveOccurred())
 
 			By("running the test pod")
-			if data, err := yaml.Marshal(pod); err != nil {
-				klog.Infof("Pod:\n%s", data)
-			}
+			data, err := yaml.Marshal(pod)
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
Marshaling should not be failed here since the pod data is hard-coded in the test.
IMVHO this is usually good to fail here.
It might reveal some human error in case of refactoring of the tests in the future.

Reply:
yes, let's keep it. I'm not sure it should be buried in here, but OTOH probably not worth a separate PR.